integration/.golangci-new.yml

```diff
@@ -18,6 +18,7 @@ linters-settings:
     exclude-functions:
       - (*database/sql.Rows).Close
       - (*github.com/FerretDB/FerretDB/internal/util/fsql.Rows).Close
+      - (*go.mongodb.org/mongo-driver/mongo.Cursor).Close
```

Comment:
Why do we exclude that?

Reply:
@noisersup `errcheck` linter throws error for unhandled return value even in `defer` statements, on statement such as,
https://github.com/FerretDB/FerretDB/blob/8edda5518ed210fcb92ee4850434381fab90d981/integration/update_compat_test.go#L246-L251

Though we have such statements all over the integration tests, I faced errors only on my changes in `testUpdateManyCompat`. After investigating, I found out that we only run linters on changed code. https://github.com/FerretDB/FerretDB/blob/8edda5518ed210fcb92ee4850434381fab90d981/integration/.golangci-new.yml#L162

IMO, it is safe to exclude this unhandled error. Otherwise, we change the defer statements as shown below, in many places of the codebase, just to keep the linter happy.
```golang
defer func() { _ = targetCursor.Close() }
```
There is no option in the `errcheck` tool to ignore unhandled errors from all `defer` statements. We have to ignore the functions one by one, as I have done.